src/datachain/cli/commands/storage/local.py

```diff
@@ -0,0 +1,127 @@
+from typing import TYPE_CHECKING
+
+from datachain.cli.commands.storage.base import StorageImplementation
+from datachain.cli.commands.storage.utils import build_file_paths, validate_upload_args
+from datachain.error import DataChainError
+
+if TYPE_CHECKING:
+    from datachain.client.fsspec import Client
+
+
+class LocalStorageImplementation(StorageImplementation):
+    def upload_to_remote(self, source_cls: "Client", destination_cls: "Client"):
+        from tqdm import tqdm
```

Comment:
can we use Datachain for all operations? instead of implementing these manually?

Datachain has read_strorage ... to_storage ... we should be able to utilize those ... otherwise we keep reimplementing the safe stuff again and again and again

Reply:
Sure, I will look into that.